elastic/kibana
# [Uptime] Add monitor list status tooltip

## Summary

Today, it is difficult for the user to determine the exact moment that their most recent check was received on the overview page. We could display a raw string but this wastes space and is less aesthetically pleasing.

A potential alternative is to add a tooltip to the text. If the user hovers over the relative time value we display, they'll be shown a localized absolute date/time string.

![image](https://user-images.githubusercontent.com/18429259/57469904-ae166280-7255-11e9-98bc-97aa1078e30a.png)

## Testing this PR

1. Start Heartbeat
2. Load Uptime UI
3. Check the tooltip
4. Stop Heartbeat
5. Make sure that the timestamp and relative string remain as you'd expect them to be

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
